diff --git a/Models.py b/Models.py
index b35ecd7..cd93f21 100644
--- a/Models.py
+++ b/Models.py
@@ -107,7 +107,7 @@ class AE_Encoder(nn.Module):
             ae_file = os.sep.join([ae_dir, 'tcra_ae_dim_' + str(self.encoding_dim) + '.pt'])
         elif self.tcr_type == 'beta':
             ae_file = os.sep.join([ae_dir, 'tcrb_ae_dim_' + str(self.encoding_dim) + '.pt'])
-        checkpoint = torch.load(ae_file)
+        checkpoint = torch.load(ae_file, map_location='cpu')
         self.autoencoder.load_state_dict(checkpoint['model_state_dict'])
         if train_ae is False:
             for param in self.autoencoder.parameters():
